target/linux/ramips/files/drivers/net/ethernet/ralink/mtk_eth_soc.c

```diff
@@ -1401,7 +1401,11 @@ static int __init fe_init(struct net_device *dev)
 	fe_reset_phy(priv);
 
 	mac_addr = of_get_mac_address(priv->dev->of_node);
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5,2,0))
 	if (mac_addr)
```

Comment:
You could use ´IS_ERR_OR_NULL()´ here, but I do not know if it is better.

Reply:
That's great! I found some usages of `IS_ERR_OR_NULL` right after `of_get_mac_address` in the kernel, so I guess it's okay to use it here as well.